enginetest/queries/queries.go

```diff
@@ -11370,3 +11370,19 @@ var IndexPrefixQueries = []ScriptTest{
 		},
 	},
 }
+
+var EnumQueries = []ScriptTest{
+	{
+		Name: "enum testing",
```

Comment:
A slightly more specific name would probably be good here... especially if we keep adding more tests here... maybe something like this? 

```suggestion
		Name: "enums in case operator",
		// https://github.com/dolthub/dolt/issues/4716
```

Reply:
Ended up removing these changes.